app/index.deck

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html class="no-js" ng-app="netflix.spinnaker">
+<html class="no-js" >
```

Comment:
Why the removal of `ng-app`?

Reply:
Never mind, I scrolled down.

Reply:
I did that based on this Stackoverflow post: https://stackoverflow.com/a/35250866

They said that Angularjs uses that to bootstrap it self and with the promise I was running into the issue where angularjs could not load the application.